config/samples/applicationMonitoring.yaml

```diff
@@ -137,3 +137,8 @@ spec:
     #     [connectivity]
     #     networkZone=
     #   valueFrom: myCustomPropertiesConfigMap
+
+    # Optional: the name of a secret containing ActiveGate TLS cert+key and password. If not set, self-signed certificate is used.
+    # server.p12: certificate+key pair in pkcs12 format
+    #
+    # tlsSecretName: "my-tls-secret"
```

Comment:
please move the samples into a separate PR as it's not connected to the logic change :)